spec/models/crime_application_spec.rb

```diff
@@ -181,4 +181,15 @@
       end
     end
   end
+
+  describe '#passporting_benefit_complete?' do
```

Comment:
I only really added this as coverage was complaining.. not sure if I need to refactor a test so the line is reached as I suspect it's because of the use of doubles